packages/flutter_test/lib/src/test_compat.dart

```diff
@@ -34,6 +34,7 @@ Declarer get _declarer {
     Future<void>(() {
       Invoker.guard<Future<void>>(() async {
         final _Reporter reporter = _Reporter(color: false); // disable color when run directly.
+        // ignore: recursive_getters
```

Comment:
Could you leave a comment explaining why this ignore, see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#comment-all--ignores

Reply:
Done.